mcurl.cpp

```diff
@@ -575,10 +575,8 @@ int MCurl::multi_timer_cb(CURLM *, long timeout_ms, MCurl *owner)
 {
```

Comment:
```
WARNING: do not call libcurl directly from within the callback itself when the timeout_ms value is zero, since it risks triggering an unpleasant recursive behavior that immediately calls another call to the callback with a zero timeout... 
```
https://curl.se/libcurl/c/CURLMOPT_TIMERFUNCTION.html